chainer/links/normalization/layer_normalization.py

```diff
@@ -0,0 +1,53 @@
+from chainer import initializers
+from chainer import link
+
+from chainer.functions.array import broadcast
+from chainer.functions.math import sqrt
+from chainer.functions.math import square
+from chainer.functions.math import sum
+from chainer.links.connection import scale
+
+
+class LayerNormalization(link.Chain):
+
+    """Layer normalization layer on outputs of linear functions.
+
+    This is a link of "Layer Normalization". This layer
+    normalizes input units by statistics along the second axis,
+    scales and shifts them with :class:`~chainer.links.Scale`.
```

Comment:
Is it required that the input to this link be a two-dimensional array, such as the output of a linear layer? Can it be applied to a convolutional layer? (although, in the paper it seems it was claimed that layer normalization does not work so well with convolutional layers anyway). If it should only be applied to the output of a linear layer, consider mentioning that.